tests/js/spec/views/organizationDiscover/discover.spec.jsx

```diff
@@ -153,14 +162,68 @@ describe('Discover', function() {
     });
 
     it('does not paginate on aggregate', async function() {
+      wrapper.instance().reset();
       wrapper.instance().updateField('aggregations', [['count()', null, 'count']]);
       wrapper.instance().runQuery();
       await tick();
       wrapper.update();
+
       expect(wrapper.find('Pagination').exists()).toBe(false);
     });
   });
 
+  describe('Pagination Subtexts', function() {
+    let wrapper;
+
+    beforeEach(function() {
+      wrapper = mount(
+        <Pagination
+          baseQuery={{
+            previous: null,
+            current: '0:0:1',
+            next: '0:10:0',
+            data: {data: [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]},
+          }}
+          getNextPage={() => jest.fn()}
+          getPreviousPage={() => jest.fn()}
+        />,
+        TestStubs.routerContext()
+      );
+    });
+
+    it('shows correct number results shown on current page', async function() {
+      expect(wrapper.find('NumberResultsShown').exists()).toBe(true);
+
+      expect(wrapper.find('NumberResultsShown').text()).toBe('Results 1 - 10');
+    });
+
+    it('shows correct number of results shown when going to next page', async function() {
```

Comment:
Might be worth adding assertions that the getNextPage/getPreviousPage mock functions are called on click